velox/functions/prestosql/aggregates/tests/ApproxPercentileTest.cpp

```diff
@@ -273,5 +273,24 @@ TEST_F(ApproxPercentileTest, partialFull) {
   exec::test::assertQuery(params, {expected});
 }
 
+TEST_F(ApproxPercentileTest, finalAggregateAccuracy) {
```

Comment:
I wonder if it would be simpler to use testAggregations helper method from the AggregationTestBase base class.

Reply:
In `testAggregations` we only pass 1 source to `localPartitionRoundRobin`, which is not enough to reproduce this.